nemo_skills/evaluation/metrics/utils.py

```diff
@@ -43,3 +42,32 @@ def is_correct_judgement(judgement):
         return False  # improper judgement format, so have to judge as false
     verdict = judgement.split('Judgement:')[-1].strip()
     return verdict.lower() == 'yes'
+
+
+def get_metrics(metric_type: str):
+
+    from nemo_skills.evaluation.metrics.answer_judgement_metrics import AnswerJudgementMetrics
```

Comment:
Let's move imports on top, no reason to hide them here

Reply:
It's not about hiding it's avoiding cross importing

Reply:
where does cross importing come from? We should fix that, it's not a good idea to have such import structure and probably indicates something else is not right in our code

Reply:
Done